button_shutdown_1       : 5  
button_shutdown_2       : 6  

button_move_enable      : 1  
axis_vel_surge          : 1  
axis_vel_sway           : 0  
axis_vel_angular        : 3  

button_kick_enable      : 2  
button_kick_straight    : 0  
button_kick_chip        : 3  
axis_kick_power         : 4  

button_dribble_enable   : 4  
axis_dribble_power      : 4  

button_id_enable        : 6 
axis_id_change          : 4 

button_color_enable     : 5 
axis_color_change       : 4 

button_all_id_1         : 1 
button_all_id_2         : 2 
button_all_id_3         : 4 
button_all_id_4         : 6 

# for indirect control
button_path_enable      : 5 
button_add_pose         : 2 
button_delete_path      : 1 
button_send_target      : 0 
